.circleci/config.yml

```diff
@@ -35,8 +34,16 @@ commands:
           name: Install build dependencies
           command: |
             yum install -y epel-release
+            if [ `rpm -E %{rhel}` = 8 ]; then
+                dnf install -y 'dnf-command(config-manager)'
+                dnf config-manager --set-enabled PowerTools
+                PSPHINX=python3-sphinx
+            else
+                PSPHINX=python-sphinx
+            fi
```

Comment:
Why do you need this? For our packaging we use `BuildRequires: python3-sphinx` for both el7 and el8.